coco/endpoint.py

```diff
@@ -40,32 +39,10 @@ def __init__(self, name, conf, slacker, forwarder, state):
         self.set_state = conf.get("set_state", None)
         self.schedule = conf.get("schedule", None)
         self.forward_checks = dict()
-        call = conf.get("call", None)
-        if call is None:
-            self.forward_name = [self.name]
-            self.forward_to_coco = None
-        else:
-            self.forward_to_coco = call.get("coco", None)
-            if self.forward_to_coco and not isinstance(self.forward_to_coco, list):
-                self.forward_to_coco = [self.forward_to_coco]
 
-            # Load name(s) of forward call endpoint(s)
-            self.forward_name = call.get("forward", [self.name])
-            # could be a string or list(str):
-            if self.forward_name and isinstance(self.forward_name, str):
-                self.forward_name = [self.forward_name]
-            # could also be a block where there are checks configured for each forward call
-            elif self.forward_name and isinstance(self.forward_name, dict):
-                for name, checks in self.forward_name.items():
-                    self.forward_checks[name] = checks
-                self.forward_name = list(self.forward_name.keys())
-
-        if self.group is None and self.forward_name:
-            logger.error(
-                f"coco.endpoint: endpoint '{name}' is missing config option 'group'. Or it "
-                f"needs to set 'call: forward: null'."
-            )
-            exit(1)
+        # To hold forward calls: first external ones than internal (coco) endpoints.
+        self.has_external_forwards = False
+        self.forwards = self._load_forwards(conf.get("call", None))
```

Comment:
this may be nitpicking, but I find it a bit confusing that `_load_forwards` sets some of the class members internally (like `forward_checks` and `has_external_forwards`) but returns a list of `forwards` when it could also just assign it to `self.forwards`. But this is really not that important, feel free to ignore.